stock_outgoing_shipment_report/report/stock_outgoing_shipment_report.py

```diff
@@ -0,0 +1,112 @@
+# Copyright 2020 Quartile Limited
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+from odoo import _, api, fields, models
+from odoo.exceptions import ValidationError
+
+FIELDS_PROPERTIES = {
+    "shipping_ref_code": ["Char", 2],
+    "carrier_name": ["Char", 20],
+    "partner_ref": ["Char", 10],
+    "partner_name": ["Char", 32],
+    "partner_zip": ["Char", 8],
+    "partner_address": ["Char", 80],
+    "product_code": ["Char", 7],
+    "product_name": ["Char", 32],
+    "case_qty": ["Float", 5],
+    "separate_qty": ["Float", 5],
+    "lot_num": ["Float", 6],
+    "lot_branch_num": ["Float", 2],
+    "delivery_division": ["Char", 1],
+    "customer_delivery_note": ["Char", 9],
+    "client_order_ref": ["Char", 30],
+    "memo": ["Char", 9],
+}
+
+
+class StockOutgoingShipmentReport(models.TransientModel):
+    _name = "stock.outgoing.shipment.report"
+
+    move_id = fields.Many2one("stock.move", string="Stock Move", readonly=True,)
+    delivery_date = fields.Char(
+        string="Delivery Date", compute="_compute_date_fields", store=True
+    )
+    expected_date = fields.Char(
```

Comment:
```suggestion
    delivery_date = fields.Char(
```
I think this should be the 'delivery' date for 納品日.